noir-projects/aztec-nr/aztec/src/encrypted_logs/log_assembly_strategies/default_aes128/note.nr

```diff
@@ -167,7 +98,7 @@ pub unconstrained fn compute_log_unconstrained<Note, let N: u32>(
 where
     Note: NoteType + Packable<N>,
 {
-    compute_log(context, note, storage_slot, recipient, sender)
+    compute_log(note, storage_slot, recipient, sender)
```

Comment:
The context was passed in only because of the contract address for the header and is now unnecessary.